client/client.go

```diff
@@ -127,3 +127,42 @@ func (client *Client) SysInfo() (*SysInfo, error) {
 
 	return &sysInfo, nil
 }
+
+// Capability holds information about a capability that a snap may request
+// from a snappy system to do its job while running on it.
+type Capability struct {
+	// Name is a key that identifies the capability. It must be unique within
```

Comment:
I'd cut back on the in-struct docs; you're duplicating the comments from `caps`, and the intended audience of client won't see this. Instead, document them in `rest.md`.